src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
Hmm, my bad. It was merged in 2.3 indeed.

However, and this is what @stof also meant, it's not deprecated in 2.3.0. It's going to be deprecated in 2.3.24 (the next symfony 2.3.x release). So the message should actually say:

``@deprecated since version 2.3.24, to be removed in 3.0.``